cypress-tests/cypress/support/redirectionHandler.js

```diff
@@ -48,27 +50,53 @@ function bankTransferRedirection(
   redirection_url,
   expected_url,
   connectorId,
-  payment_method_type
+  payment_method_type,
+  next_action_type
 ) {
-  cy.request(redirection_url.href).then((response) => {
-    switch (connectorId) {
-      case "adyen":
-        switch (payment_method_type) {
-          case "pix":
-            expect(response.status).to.eq(200);
-            fetchAndParseQRCode(redirection_url.href).then((qrCodeData) => {
-              expect(qrCodeData).to.eq("TestQRCodeEMVToken");
-            });
+  switch (next_action_type) {
+    case "qr_code_url":
+      cy.request(redirection_url.href).then((response) => {
+        switch (connectorId) {
+          case "adyen":
+            switch (payment_method_type) {
+              case "pix":
+                expect(response.status).to.eq(200);
+                fetchAndParseQRCode(redirection_url.href).then((qrCodeData) => {
+                  expect(qrCodeData).to.eq("TestQRCodeEMVToken");
+                });
+                break;
+              default:
+                verifyReturnUrl(redirection_url, expected_url, true);
+              // expected_redirection can be used here to handle other payment methods
+            }
             break;
           default:
             verifyReturnUrl(redirection_url, expected_url, true);
-          // expected_redirection can be used here to handle other payment methods
         }
-        break;
-      default:
-        verifyReturnUrl(redirection_url, expected_url, true);
-    }
-  });
+      });
+    break;
+    case "image_data_url":
+      switch (connectorId) {
+        case "itaubank": 
+          switch (payment_method_type) {
+            case "pix":
+              fetchAndParseImageData(redirection_url).then((qrCodeData) => {
+                console.log("Imaage_data>>>>>>>>>", qrCodeData)
```

Comment:
`console.log()` can be removed as it is no longer necessary.

Reply:
Since these are minor changes, Im putting them in the subsequent pr for refund